### ISSUE TITLE
A brand-new list

### ISSUE THREAD

**Issue by GrimPixel** [Sw24Softwares/StarkeVerben#26]
Do you think you need this list? You can freely redistribute it.
https://polyglotclub.com/wiki/Language/German/Vocabulary/Common-strong-and-mixed-verbs

**Comment by 24PaH** [comment]
Yeah, we will probably need to do this one day :D. Thank you for this link
